src/styles/dictionary_modal.scss

```diff
@@ -147,11 +147,7 @@ legend {
       color: #cc0000;
   }
 
-button:disabled{
```

Comment:
@judeinno just asking. Won't this style be needed somewhere else in the application?

Reply:
We are using the default disabled for consistency.